micronaut-runtime-groovy version is incorrect in BOM

4.0.0 and newer have an incorrect version of micronaut-runtime-groovy in the Grails BOM.

micronaut-runtime-groovy only has a version up to 1.1.1 and so Gradle cannot resolve the dependency. 

The BOM should specify a real version of this dependency.

### Task List

- [ x ] Steps to reproduce provided
- [ x ] Stacktrace (if present) provided
- [ x ] Example that reproduces the problem uploaded to Github
- [ x ] Full description of the issue provided (see below)

### Steps to Reproduce

1. Create new Grails App (4.0.0 or 4.0.1)
2. Add dependency on micronaut-runtime-groovy without version
3. Run any gradle task

### Expected Behaviour

Gradle should be able to configure itself and run the task

### Actual Behaviour

Gradle attempts to find `io.micronaut:micronaut-runtime-groovy:1.1.4` instead of a version that exists, and therefore fails

### Stacktrace
```
Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'test'.
> Could not resolve all files for configuration ':runtime'.
   > Could not find io.micronaut:micronaut-runtime-groovy:1.1.4.
     Searched in the following locations:
       - https://repo.grails.org/grails/core/io/micronaut/micronaut-runtime-groovy/1.1.4/micronaut-runtime-groovy-1.1.4.pom
       - https://repo.grails.org/grails/core/io/micronaut/micronaut-runtime-groovy/1.1.4/micronaut-runtime-groovy-1.1.4.jar
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 38s
```

### Environment Information

- **Operating System**: Windows 10 1903
- **Grails Version:** 4.0.0
- **JDK Version:** 1.8

### Example Application

https://github.com/jjelliott/grails-bom-micronaut-issue

--- a/build.gradle
+++ b/build.gradle
@@ -43,7 +43,13 @@ ext {
                 version: micronautVersion,
                 group:'io.micronaut',
                 names:['micronaut'],
-                modules:['spring','inject','runtime', 'inject-groovy', 'runtime-groovy', 'inject-java', 'core', 'http-client', 'http', 'buffer-netty', 'http-netty']
+                modules:['spring','inject','runtime', 'inject-groovy', 'inject-java', 'core', 'http-client', 'http', 'buffer-netty', 'http-netty']
+        ],
+        'micronaut.micronaut-runtime-groovy': [
+                version: micronautRuntimeGroovyVersion,
+                group:'io.micronaut',
+                names:['micronaut'],
+                modules:['runtime-groovy']
         ],
         'micronaut.spring': [
                 version: micronautSpringVersion,

--- a/gradle.properties
+++ b/gradle.properties
@@ -4,6 +4,7 @@ springVersion=5.1.12.RELEASE
 spockVersion=1.2-groovy-2.5
 micronautVersion=1.1.4
 micronautSpringVersion=1.0.2
+micronautRuntimeGroovyVersion=1.1.1
 springLoadedVersion=1.2.8.RELEASE
 gradleSdkVendorVersion=1.1.1
 jaxbVersion=2.3.1

